### ISSUE TITLE
[Support Request] Sync scx to remote HEAD

### ISSUE THREAD

**Issue by yqlbu** [chaotic-cx/nyx#646]
### What happens?

Recently, I met an issue where the `scx` program cannot work properly on my laptop with `meteor lake` CPU. It turns out that it has something to do with the recent changes the contributors made in the [scx](https://github.com/sched-ext/scx) repo.

More context information available at https://github.com/sched-ext/scx/issues/195

### What is expected to happen?

Expect `scx` program works properly with `meteor lake` CPU.

### If possible, please attach logs

```console
⯁ ~ ❯❯❯ grep . /sys/devices/system/node/node*/cpu*/cache/index*/id
fish: No matches for wildcard '/sys/devices/system/node/node*/cpu*/cache/index*/id'. See `help wildcards-globbing`.
grep . /sys/devices/system/node/node*/cpu*/cache/index*/id
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
```

### More information

The contributor already fixed the issue, so requesting to sync upstream HEAD to include the fixes. Thanks in advance.

**Comment by PedroHLC** [comment]
Sorry, I won't be in front of a proper computer till Monday

**Comment by yqlbu** [comment]
> Sorry, I won't be in front of a proper computer till Monday

No hurry at all. Thanks for you kind reply in https://github.com/sched-ext/scx/issues/195.

**Comment by PedroHLC** [comment]
@yqlbu, do think you can push a PR? You should be able to do it by updating `pkgs/scx/common.nix` and all the `Cargo.lock` inside `pkgs/scx` (running `cargo generate-lockfile` inside each directory within a clone of the original repo).

**Comment by yqlbu** [comment]
> @yqlbu, do think you can push a PR? You should be able to do it by updating `pkgs/scx/common.nix` and all the `Cargo.lock` inside `pkgs/scx` (running `cargo generate-lockfile` inside each directory within a clone of the original repo).

Hi Pedro, actually I've made an attempt but no luck - failed in the compilation step. I have no clue what is going on TBH.

**Comment by PedroHLC** [comment]
Solved in #655

**Comment by yqlbu** [comment]
> Solved in #655

Thanks for the kind help man!
